server/main.py

```diff
@@ -25,19 +34,40 @@ class Post(BaseModel):
     password:str
     key:bytes
 
+
+
 @app.get("/")
 async def root():
     return "welcome to the server made by ravindra kumar saini"
+
+
+
+
+
+
 @app.post("/get_data/{username}")
-async def get_data(username:str,new:Get):
+async def get_data(username:str,new:Get,response:Response):
+    password = new.password
     username = username
-    database = Database('sqlite+aiosqlite:///database.db')
-    await database.connect()
-    rows = await database.fetch_one("select * from user where username=:username and password=:password;",{"username":username,"password":new.password})
-    if rows is None:
-        raise HTTPException(403,"No such username exits here")
-    else:
-        return dict(rows)["key"]
+    async with Database("sqlite+aiosqlite:///database.db") as database:
+        async with database.transaction():
+            row = await database.fetch_one("select key from user where username=:username and password=:password;",{"username":username,"password":password})
+            row = dict(row)
+            if row == 0:
+                raise HTTPException(403,f"no username found {username}")
+            else:
+                response.set_cookie(key="cookie",value=await set_otp(username,password))
```

Comment:
Give a descriptive name to a cookie. Like a session_token or something. There can be multiple cookies with different names (for different purposes)